mycore-base/src/main/java/org/mycore/common/MCRSession.java

```diff
@@ -153,14 +158,16 @@ public void close() {
         });
     }
 
+
     /**
      * The constructor of a MCRSession. As default the user ID is set to the value of the property variable named
      * 'MCR.Users.Guestuser.UserName'.
      */
     MCRSession() {
         userInformation = guestUserInformation;
         setCurrentLanguage(MCRConfiguration2.getString("MCR.Metadata.DefaultLang").orElse(MCRConstants.DEFAULT_LANG));
-        dataBaseAccess = MCRHIBConnection.isEnabled();
+        dataBaseAccess = MCRConfiguration2.getBoolean("MCR.Persistence.Database.Enable").orElse(true)
```

Comment:
There should be a utility method / class for this essential check and not just only a "low-level" property lookup

Reply:
It's just reading a property with a default value and you need an extra method for it?